Workbooks/PostgreSQL Flexible Server/Performance/High CPU Usage/High CPU Usage.workbook

```diff
@@ -3142,6 +3272,548 @@
+                  "subTarget": "mean",
+                  "style": "link"
+                },
+                {
+                  "id": "a2e173a2-5be3-46a5-8a25-b49022c8d474",
+                  "cellValue": "logmindurationtab",
+                  "linkTarget": "parameter",
+                  "linkLabel": "Queries by calls",
+                  "subTarget": "calls",
+                  "style": "link"
+                }
+              ]
+            },
+            "name": "logmindurationtabs"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "let graph_grain = iff({TimeRange:grain} < 5m, 5m, {TimeRange:grain});\r\nlet Start_Time = todatetime('{GlobalStartTime}');\r\nlet End_Time = todatetime('{GlobalEndTime}');\r\nlet TopN = 25;\r\n\r\nlet base_logs = \r\n    {PGServerLogsTable}\r\n    | where TimeGenerated between (Start_Time .. End_Time)\r\n    | where Message has \"duration\"\r\n    | where Message !contains 'bind' and Message !contains 'parse'\r\n    | parse Message with * \"duration: \" duration_ms:double \" ms\" *\r\n    | extend statement = coalesce(\r\n      extract(@\"statement:\\s*((.|\\n)*)\", 1, Message),\r\n      extract(@\"execute\\s+[^\\s]+:\\s*((.|\\n)*)\", 1, Message))\r\n    | where isnotempty(statement)\r\n    | extend database = extract(\"db=([^,]+)\", 1, Message)\r\n    | extend application = extract(\"app=([^ ,]+)\", 1, Message)\r\n    | extend user = extract(\"user=([^,]+)\", 1, Message)\r\n    | where database !in('azure_sys', 'azure_maintenance')\r\n    | where user!=\"azuresu\"\r\n    | where duration_ms > {duration}\r\n    | project TimeGenerated, statement, duration_ms, database, application, user;\r\n\r\nlet summary = \r\n    base_logs\r\n    | summarize \r\n        calls = count(), \r\n        total_duration_ms = round(sum(duration_ms), 2), \r\n        avg_duration_ms = round(avg(duration_ms), 2),\r\n        p95_duration_ms = percentile(duration_ms, 95)\r\n      by statement, database, application, user;\r\n\r\nlet trends = \r\n    base_logs\r\n    | make-series \r\n        call_count = count(), \r\n        total_duration = sum(duration_ms),\r\n        p95_duration_trend = percentile(duration_ms, 95)\r\n        on TimeGenerated from Start_Time to End_Time step graph_grain\r\n        by statement, database, application, user;\r\n\r\ntrends\r\n| join kind=inner (summary) on statement, database, application, user\r\n| project statement, database, application, user, calls, calls_trend=call_count, total_duration_ms, total_duration_trend=total_duration, avg_duration_ms, p95_duration_ms, p95_duration_trend\r\n| order by total_duration_ms desc\r\n| serialize\r\n| extend rank = row_number()\r\n| project rank, statement, database, application, user, calls, total_duration_ms, avg_duration_ms, p95_duration_ms,calls_trend,total_duration_trend, p95_duration_trend\r\n| limit TopN\r\n\r\n",
```

Comment:
Column names shouldn't include the unit embedded in the name. Also, word separator should be spaces, not underscores. Therefore, **total_duration_ms** should be **total duration**, **p95_duration_ms** should be **p95 duration**, etc.